pkg/daemon/daemon.go

```diff
@@ -42,6 +42,16 @@ type ptpProcess struct {
 	cmd             *exec.Cmd
 }
 
+type ptp4lconfSection struct {
+	name            string
+	options          []string
+}
+
+type ptp4lconf struct {
+	sections        map[string]ptp4lconfSection
+	ifaceString     string
+}
```

Comment:
can you declare these and the relevant methods in config.go?

Reply:
Will update in next revision.